test/cases/parsing/issue-4940/index.js

```diff
@@ -0,0 +1,5 @@
+it("should create dependency when require is called with 'new'", function() {
+	const sideEffect = require("./sideEffect");
+	new require("./module");
```

Comment:
Could you also check for the correct return value?

Reply:
What do you consider the correct return value?

Reply:
The same as `require`

Reply:
I don't think that's possible